### PR TITLE
temporary dismiss

### PR DESCRIPTION
**Pull request by elizzhao**
Enrolled banner can be dismissed for the session. Will reappear on refresh and on closing then reopening the page.

### CODE REVIEW

File: src/components/TrialBanner.js [hunk 1]
```diff
@@ -156,6 +157,11 @@ export const TrialBanner = _.flow(
           button.isExternal ? icon('pop-out', { style: { marginLeft: '0.25rem' } }) : null
         ]),
         div({ style: { marginLeft: '3rem', display: 'flex', flexDirection: 'column', alignItems: 'flex-end' } }, [
+          (trialState === 'Enrolled') && h(Clickable, {
+            style: { borderBottom: 'none' },
+            tooltip: 'Hide for now',
```
Comment (katevoss) [inline src/components/TrialBanner.js]: Maybe clarify to "Hide banner" or "Hide banner temporarily" to describe the exact effect of the users action?

**Review by katevoss — APPROVED** [review]
Works and looks good! One comment on tooltip.

File: src/components/TrialBanner.js (hunk 1 above)
Comment (elizzhao) [reply src/components/TrialBanner.js]: Adding the temporarily makes the tool tip kinda long, so I'll change it to "Hide Banner"

File: src/components/TrialBanner.js [hunk 2]
```diff
@@ -156,6 +157,11 @@ export const TrialBanner = _.flow(
           button.isExternal ? icon('pop-out', { style: { marginLeft: '0.25rem' } }) : null
         ]),
         div({ style: { marginLeft: '3rem', display: 'flex', flexDirection: 'column', alignItems: 'flex-end' } }, [
+          (trialState === 'Enrolled') && h(Clickable, {
+            style: { borderBottom: 'none' },
+            tooltip: 'Hide banner',
+            onClick: () => this.setState({ snoozeBanner: true })
+          }, [icon('times-circle', { size: 25, style: { fontSize: '1.5rem', cursor: 'pointer', strokeWidth: 1.5 } })]),
```
Comment (breilly2) [inline src/components/TrialBanner.js]: Nit: I don't think `strokeWidth` is doing anything to the icon (which is unfortunate). Should remove it for clarity.

**Review by breilly2 — APPROVED** [review]
👍 One nit but otherwise good

**Pull request merged**